backend/H2Oasis.Api/Services/HouseholdService.cs

```diff
@@ -71,6 +71,16 @@ public async Task<Household> CreateHousehold(Household newHousehold)
         _dbContext.Entry(existingHousehold).CurrentValues.SetValues(updatedHousehold);
         await _dbContext.SaveChangesAsync();
         return existingHousehold;
+        
+       
+        // var updatedHouseholdWithDetails = await _dbContext.Households
+        //     .Where(h => h.HouseholdId == updatedHousehold.HouseholdId)
```

Comment:
Will this be used? as I see it's commented out. As well, could we use === instead of ==?

Reply:
Yeah we actually dont need it. I'll remove it :)